src/main/java/com/mparticle/kits/GoogleAnalyticsFirebaseKit.java

```diff
@@ -303,7 +311,7 @@ public void onRemoveUserAttribute(String key, FilteredMParticleUser filteredMPar
     @Override
     public void onSetUserAttribute(String key, Object value, FilteredMParticleUser filteredMParticleUser) {
         if (value instanceof String) {
-            FirebaseAnalytics.getInstance(getContext()).setUserProperty(key, (String)value);
+            FirebaseAnalytics.getInstance(getContext()).setUserProperty(standardizeName(key, false), standardizeValue((String)value, false));
```

Comment:
value isn't always a string. Will this crash?

Reply:
checking it in the previous line, so this won't hit if it's not a String

Reply:
Oh but non-string values are acceptable to Firebase

Reply:
yea, so I just had to look back at this, since our public API signature is an object `MParticleUser.setUserAttribute(key: String, value: Object)`. The first step we do "stringify" everything after it's passed in, so the `Object` type of the "value" parameter isn't actually a 1 to 1 representation of our API method, but actually a catch all for `setUserAttribute(String, String)`/`setUserAttributeList(String, List<String>)`, and the value here will only every be a `String` or a `List<String>`.

So, the Firebase method is looking for a `String`, but if a client passes into the public API `setUserAttribute("key", 5)`, this method will ultimately be called with `setUserAttribute("key", "5")` anyway